plugins/theme-modern/README.md

```diff
@@ -0,0 +1,30 @@
+# @blockly/theme-modern [![Built on Blockly](https://tinyurl.com/built-on-blockly)](https://github.com/google/blockly)
+
+A [Blockly](https://www.npmjs.com/package/blockly) modern theme.
```

Comment:
Maybe a bit more description here? " A Blockly theme that has the same classic colors with no highlight" Or something like that.

Reply:
Yeah I struggled to come up with a description of the modern theme.
Same classic colours but sets the secondary and tertiary colours of blocks, but would developers really know what that entails?

Dark / light paths are controlled by the renderer, in geras this would still have these paths, we are just controlling the colours of these paths with secondary / tertiary colours.

Thoughts on what this could say?

Reply:
You could  just add a [link](https://developers.google.com/blockly/guides/configure/web/themes#block_style) to the description of the secondary and tertiary theme. 

Or you could say something like "A modern theme has the same classic colours, but with a darker border. This is mainly meant for use with thrasos or zelos renderer." Added the last bit since it doesn't look great with geras but up to you whether you want to include.

Reply:
Updated.